poetry.lock

```diff
@@ -31,6 +31,29 @@ python-versions = ">=3.6"
 [package.dependencies]
 pytz = ">=2015.7"
 
+[[package]]
```

Comment:
are you supposed to commit the lock file? This was contentious for Pipenv in its time